packages/core/src/editor/BlockNoteEditor.ts

```diff
@@ -777,7 +777,6 @@ export class BlockNoteEditor<
     for (const mark of marks) {
       const config = this.schema.styleSchema[mark.type.name];
       if (!config) {
-        console.warn("mark not found in styleschema", mark.type.name);
```

Comment:
This warning is still valid I think? Maybe we need to just exclude the case for Links? (because links don't appear in the styleschema but are an IC node in blocknote)